src/slider/tp-slider.ts

```diff
@@ -171,7 +171,10 @@ export class TPSliderElement extends HTMLElement {
 		this.updateHeight();
 
 		// Now lets slide!
-		slidesContainer.style.left = `-${ this.offsetWidth * ( this.currentSlideIndex - 1 ) }px`;
+		const behaviour = this.getAttribute( 'behaviour' ) || '';
```

Comment:
@imranhsayed Lets be strict: `const behaviour: string = this.getAttribute( 'behaviour' ) || ''`